app/controllers/FeedbackFormsResponseController.scala

```diff
@@ -104,4 +150,42 @@ class FeedbackFormsResponseController @Inject()(messagesApi: MessagesApi,
             new Date(session.expirationDate.value).toString))
       }
 
+  def storeFeedbackFormResponse: Action[JsValue] = userAction.async(parse.json) { implicit request =>
+    request.body.validate[FeedbackResponse].asOpt.fold {
+      Logger.error(s"Received bad request while storing feedback response, ${request.body}")
+      Future.successful(BadRequest("Malformed Data!"))
+    } { feedbackFormResponse =>
+      val validatedForm =
+        feedbackFormResponse.validateSessionId orElse
+          feedbackFormResponse.validateForm orElse feedbackFormResponse.validateQuestion orElse
+          feedbackFormResponse.validateOptions orElse feedbackFormResponse.validateFormResponse
+
+      validatedForm.fold {
+        val questionAndResponseInformation =
+          feedbackFormResponse.questionsAndResponses.map(responseInformation =>
+            QuestionResponse(responseInformation.question, responseInformation.options, responseInformation.response))
+
+        val dateTime = new Date(System.currentTimeMillis).getTime
```

Comment:
Though we can directly use System.currentTimeMillis instead of new Date(System.currentTimeMillis).getTime, but can we use DateTimeUtility to get current time in milliseconds instead?